classes/locations.php

```diff
@@ -323,29 +328,29 @@ function additional_information( $post ) {
 
 					<!-- Phone -->
 					<tr valign="top">
-						<td width="150"><label for="location_phone"><?php _e( 'Phone', 'SimpleMap' ); ?></label></td>
+						<td width="150"><label for="location_phone"><?php _e( 'Phone', 'simplemap' ); ?></label></td>
 						<td><input type="text" id="location_phone" name="location_phone" size="30" maxlength="28"
 								   value="<?php echo esc_attr( $location_phone ); ?>"/></td>
 					</tr>
 
 					<!-- Email -->
 					<tr valign="top">
-						<td><label for="location_email"><?php _e( 'Email', 'SimpleMap' ); ?></label></td>
+						<td><label for="location_email"><?php _e( 'Email', 'simplemap' ); ?></label></td>
 						<td><input type="text" name="location_email" id="location_email" size="30"
 								   value="<?php echo esc_attr( $location_email ); ?>"/>
 					</tr>
 
 					<!-- URL -->
 					<tr valign="top">
-						<td><label for="location_url"><?php _e( 'URL', 'SimpleMap' ); ?></label></td>
+						<td><label for="location_url"><?php _e( 'URL', 'simplemap' ); ?></label></td>
 						<td><input type="text" name="location_url" id="location_url" size="30"
 								   value="<?php echo esc_attr( $location_url ); ?>"/>
-							<br/><?php _e( 'Please include <strong>http://</strong>', 'SimpleMap' ); ?></td>
+							<br/><?php _e( 'Please include <strong>http://</strong>', 'simplemap' ); ?></td>
 					</tr>
 
 					<!-- Fax -->
 					<tr valign="top">
-						<td><label for="location_fax"><?php _e( 'Fax', 'SimpleMap' ); ?></label></td>
+						<td><label for="location_fax"><?php _e( 'Fax', 'simplemap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->